src/main/resources/db/migration/V1_1__add_municipality_id.sql

```diff
@@ -0,0 +1,2 @@
+alter table if exists batch
+    add column municipality_id varchar(255);
```

Comment:
Do we need an index here or is this table cleaned  after processing? 
I.e. number of records will allways be low?)

If not, an index will probably be a good thing, since the municipalityId is used in query-criteria

Reply:
The batch record is removed after a batch is processed, so the amount of records should probably stay fairly low. But making it a index won't hurt I guess.